### ISSUE TITLE
Other gam (mgcv) families?

### ISSUE THREAD

**Issue by wfolta** [paul-buerkner/brms#175]
You've allowed the use of mgcv's gam's `s()` function. I was surprised yesterday to discover that mgcv also supports `cox.ph` and `cox.pht` families. I imagine families are a more difficult task than a function, but I've had to recently retreat from a fully-parametric survival analysis to a semi-parametric (e.g. Cox PH) approach.

Any chance of adding these two families?

**Comment by paul-buerkner** [comment]
I am surprised too. It depends on how these families are implemented under the hood and how they fit in the brms framework. I had thought about cox-regression at some point (a few weeks ago), when someone asked me a related question. The problem with cox-regression I currently see is the problem that the surivival time is the "formal" response variable, but the harzard function is the one actually regressed, which is not the typical structure assumed in brms. Do you know of any Stan / Jags / BUGS code implementing cox regression?

**Comment by wfolta** [comment]
Sorry, I haven't found any Stan/JAGS code yet. I was thinking Cox PH was an entirely different world until I stumbled onto `cox.ph`.  I'm not particularly happy working in that semi-parametric world with PH requirements, but it's not going to happen in the parametric world without something like mixture models.

(Speaking of which, I have found a paper on a parametric PH model the uses DP's to drive a mixture of Weibull's, but that's really different and I'm not sure it would fit in with your mixed models ideas and not even sure if it works in Stan -- as opposed to JAGS. I've heard about Stan having to have things marginalized out to work, but I'm not sure about all the details. http://www.stat.ncsu.edu/information/library/papers/ISMS2563.pdf)
